src/JS_Modules/Controllers/ModalHandler.js

```diff
@@ -1,8 +1,67 @@
 import { Modal } from 'bootstrap/dist/js/bootstrap.bundle.js';
+import getMoviePopupDetails from '../Models/Services.js';
 
-const commentsModalHandler = () => {
-  // add movieID to FUNCTION PARAMETER
+// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line arrow-body-style
```

Comment:
It would be nice to remove the comments and eslint-disabler. This would make the code appear better

Reply:
Done